pages/[content].tsx

```diff
@@ -13,6 +13,7 @@ import Charts from "../components/ContentTemplates/ChartsTemplate"
 import Icons from "../components/ContentTemplates/IconsTemplate"
 import Images from "../components/ContentTemplates/ImagesTemplate"
 import Video from "../components/ContentTemplates/VideoTemplate"
+import Alerts from "../components/ContentTemplates/AlertsTemplate"
```

Comment:
Could you add this import in alphabetical order too?

Reply:
@EmmaDawsonDev okay. Added the changes in the commit